app/helpers/claims_helper.rb

```diff
@@ -62,11 +62,6 @@ def misc_fees_summary_locals(claim, args = {})
     }
   end
 
-  def display_unused_materials_notice?(claim)
-    claim.eligible_misc_fee_types.map(&:unique_code).include?('MIUMU') &&
-      claim.fees.none? { |f| f.fee_type.unique_code == 'MIUMU' }
-  end
-
   def unclaimed_fees_list(claim)
```

Comment:
really nitpicky and not a must but `unclaimed_fees_list` could be called `unclaimed_misc_fees_list`